feather-ui/src/lib.rs

```diff
@@ -437,6 +444,13 @@ impl Default for RelLimits {
 }
 
 impl RelLimits {
+    #[inline]
```

Comment:
kinda annoying that all these inlines probably help somewhat when lto is off T_T